src/NHibernate/Cfg/Configuration.cs

```diff
@@ -124,7 +123,7 @@ public void GetObjectData(SerializationInfo info, StreamingContext context)
 		{
 			ConfigureProxyFactoryFactory();
 			SecondPassCompile();
-			Validate();
+			Validate(BuildMapping());
```

Comment:
Not sure if we want/need to validate configuration on serialization?

Reply:
I would say no

Reply:
I would leave it as is though.

Reply:
Why? Wouldn't it throw IOE? At least you shouldn't use obsolete method here.

Reply:
> Wouldn't it throw IOE? 

Highly likely not. 

Ok, I've removed the validation.